async-request-reply/README.md

```diff
@@ -75,7 +102,7 @@ For more information about this pattern, see [Asynchronous Request-Reply pattern
     Location: http://<appservice-name>.azurewebsites.net/api/RequestStatus/<guid>
    ```
 
-   Using a browser open the url from the *Location* field in the response. A file with the data you have sent will be downloaded.
+   Using a browser open the url from the _Location_ field in the response. A file with the data you have sent will be downloaded.
 
    > **Note** the app uses the WEBSITE_HOSTNAME environment variable. This environment variable is set automatically by the Azure App Service runtime environment. For more information, see [Azure runtime environment](https.://github.com/projectkudu/kudu/wiki/Azure-runtime-environment)
 
```

Comment:
I can't make a suggestion on line 109 (thanks GitHub UI), but for the Clean up header right below this comment, can you change it to:

`## :broom: Clean up resources`  (so, H2, and use our standard icon for the cleanup step.)

Reply:
done